components/single_image_view/single_image_view.jsx

```diff
@@ -18,7 +18,7 @@ const PREVIEW_IMAGE_MIN_DIMENSION = 50;
 
 export default class SingleImageView extends React.PureComponent {
     static propTypes = {
-        postId: PropTypes.string.isRequired,
+        post: PropTypes.object.isRequired,
```

Comment:
Instead of updating this to receive the post object as a prop, can you instead continue passing the post ID and then just pull the correct post using a redux selector in the `view_image/index.js`? It's generally better to avoid passing objects down multiple layers as props where avoidable and to pass primitives instead

Reply:
Makes sense. Updated.